pythoncalcultor/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
Looks like a typo here, I have suggested a change here
```suggestion
    return x * y
```